main/lib/src/CommandReceiver.cc

```diff
@@ -112,7 +132,12 @@ namespace eudaq {
     std::cout << "Config:\n" << param << std::endl;
   }
 
-  void CommandReceiver::OnClear() { SetStatus(Status::LVL_NONE, "Wait"); }
+  void CommandReceiver::OnClear() {
+
+    if(m_connectionstate.GetState()== eudaq::ConnectionState::STATE_RUNNING)
+      OnStopRun();
```

Comment:
When will the OnClear be called?  Can it be completly removed?
It is better to avoid calling the the OnStopRun inside the OnClear. In the case the OnXXX functions call each other, it would be problematic. (infinit circle call) . OnXXX functions are only the entrance function to handle the correlated command.